### PR TITLE
change language in dropdown on language click

### PR DESCRIPTION
**Pull request by maxhaggmancmedu**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,18 @@ import Languages from './components/languages/languages';
 import Header from './components/header';
 import Footer from './components/footer/footer';
 import SearchBar from './components/searchbar/SearchBar'
+import { useState } from 'react';
+import { languages } from './languages';
 
 function App() {
+
```
Comment (DavidDyberg) [inline src/App.js:10]: Remove this row

File: src/components/select/index.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    padding: 4px;
+    text-align: left;
+    width: fit-content;
+}
+
+.select-languages:hover {
+    background-color: #eaecf0;
+    cursor: pointer;
+}
+
```
Comment (DavidDyberg) [inline src/components/select/index.css]: Remove row 15,16 and 17

**Review by DavidDyberg — CHANGES_REQUESTED** [review]
Remove empty rows at "src/components/select/index.css"

**Comment by maxhaggmancmedu** [comment]
Empty rows have been removed now, please review again!

**Review by DavidDyberg — APPROVED** [review]

**Review by CyberGalactus — APPROVED** [review]

**Review by CyberGalactus — APPROVED** [review]

**Review by DavidDyberg — APPROVED** [review]

**Review by DavidDyberg — APPROVED** [review]

**Review by DavidDyberg — APPROVED** [review]

**Pull request merged**